clients/wallet/src/keys.rs

```diff
@@ -5,16 +5,8 @@ use std::env;
 // If one variable is not on the system environment, and the .env is defined, then all variables 
 // will be overridden by those on the .env file.
 fn get_env_variables(key: &str) -> Option<String> {
-    match env::var(key) {
-        Ok(value) => Some(value),
-        Err(_) => {
-            if dotenv::from_filename("../vault/resources/secretkey/.env").is_ok() {
-                env::var(key).ok()
-            } else {
-                None
-            }
-        }
-    }
+    dotenv::from_filename("../vault/resources/secretkey/.env").ok();
```

Comment:
Should we make this path configurable? I think we at least shouldn't put any nesting here because if you are a vault operator running the binary, you probably don't want to create this folder structure next to your binary. I guess it's best if we change the path to something simple like `"./.env"`

Reply:
The thing is that as far as I understand the keys fetched from the environment are only for tests right? Operators will supply it via flag and there is no issue them not having the keys for testing. Since we already had that folder structure for testing it won't be a problem for us, what do you think? Or is it that operators may also perform test in some scenarios?

Reply:
Oh you are totally right, I forgot about this. 

Let's still put the .env file in a higher folder level then. We can put it into the `/clients/.env` or `clients/wallet/.env`.

Reply:
Agreed, under client will be easier to handle.